projects/kit/components/input-count/input-count.component.ts

```diff
@@ -13,6 +13,7 @@ import {
 import {NgControl} from '@angular/forms';
 import {
     AbstractTuiControl,
```

Comment:
Is this still used?

Reply:
yes, in v3.x
<img width="687" alt="image" src="https://user-images.githubusercontent.com/12021443/210039479-04448b8b-0d07-4887-8351-e3b863588bbc.png">

in v2.x
<img width="508" alt="image" src="https://user-images.githubusercontent.com/12021443/210039494-6490c7d6-897b-40cd-82b2-2a05a1d28ea4.png">